apps/api/src/domain/models/audio-visual/shared/methods/import-line-items-to-transcript.ts

```diff
@@ -13,6 +12,7 @@ interface Transcribable {
     getTimeBounds(): [number, number];
 }
 
+// Should we rename this to `import-transcript`?
 // TODO Do we ensure that there must be at least one line item here?(
```

Comment:
Let's consider this.

Reply:
This could be done as part of [this story](https://www.pivotaltracker.com/story/show/187128338).